package-list.json

```diff
@@ -38,7 +47,7 @@
       "path": "http://hl7.org.au/fhir/core/0.2.0-preview",
       "fhirversion": "4.0.1",
       "status": "draft",
-      "sequence": "AU Core R4"
```

Comment:
This change is not related to adding 0.3.0-ballot, it is changing how sequencing is categorised. The current sequence approach was indicating the version of FHIR not the release of AU Core. We will need to meet on this to change how the versioning is progressing and discuss AU Base and other HL7 AU IGs.